inference-engine/tests/functional/inference_engine/ngraph_reader/mvn_tests.cpp

```diff
@@ -68,15 +80,28 @@ TEST_F(NGraphReaderTests, ReadMVNNetwork) {
                 </port>
             </output>
         </layer>
+		<layer name="Const_1" type="Const" precision="FP32" id="2">
```

Comment:
Several questions about tests:

- Could we keep tests for MVN from previous opsets?
- Why do you change the legacy IR? In legacy IR we didn't have `Const` layers and MVN layer didn't have `eps_mode` atribute.